x/gamm/keeper/multihop_test.go

```diff
@@ -95,6 +136,14 @@ func (suite *KeeperTestSuite) TestBalancerPoolSimpleMultihopSwapExactAmountIn()
 				// Ratio of the token out should be between the before spot price and after spot price.
 				sp := test.param.tokenIn.Amount.ToDec().Quo(tokenOutAmount.ToDec())
 				suite.True(sp.GT(spotPriceBefore) && sp.LT(spotPriceAfter), "test: %v", test.name)
+
```

Comment:
Can we have test cases that tests with exact numbers on reduce fees?

Reply:
Agreed, I don't think GT LT is sufficient for this